VMWare boxes for Wheezy 7.8

Would it possible to provide a vmware box for Wheezy 7.8?

With VMware Tools 7.7.x `vmhgfs` does not currently compile on Debian 7.8.  We don't like releasing boxes that do not support the guest tools fully.  This latest round of VMware Tools releases has not worked on Debian Wheezy for some reason.  If you can figure out a way to get the tools to compile, I would have no problems producing a box.

Here's the error:

```
 vmware-iso: make: Entering directory `/tmp/modconfig-VEFxrq/vmhgfs-only'
    vmware-iso: /usr/bin/make -C /lib/modules/3.2.0-4-amd64/build/include/.. SUBDIRS=$PWD     SRCROOT=$PWD/. \
    vmware-iso: MODULEBUILDDIR= modules
    vmware-iso: make[1]: Entering directory `/usr/src/linux-headers-3.2.0-4-amd64'
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/backdoor.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/backdoorGcc64.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/bdhandler.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/cpName.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/cpNameLinux.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/cpNameLite.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/dentry.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/dir.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/file.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/filesystem.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/fsutil.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/hgfsBd.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/hgfsEscape.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/hgfsUtil.o
    vmware-iso: CC [M]  /tmp/modconfig-VEFxrq/vmhgfs-only/inode.o
    vmware-iso: /tmp/modconfig-VEFxrq/vmhgfs-only/inode.c: In function ‘HgfsPermission’:
    vmware-iso: /tmp/modconfig-VEFxrq/vmhgfs-only/inode.c:1978:34: error: ‘struct dentry’ has no member named ‘d_alias’
    vmware-iso: /tmp/modconfig-VEFxrq/vmhgfs-only/inode.c:1978:34: warning: initialization from incompatible pointer type [enabled by default]
    vmware-iso: /tmp/modconfig-VEFxrq/vmhgfs-only/inode.c:1978:34: warning: (near initialization for ‘dentry’) [enabled by default]
    vmware-iso: /tmp/modconfig-VEFxrq/vmhgfs-only/inode.c:1978:34: error: ‘struct dentry’ has no member named ‘d_alias’
    vmware-iso: make[4]: *** [/tmp/modconfig-VEFxrq/vmhgfs-only/inode.o] Error 1
    vmware-iso: make[3]: *** [_module_/tmp/modconfig-VEFxrq/vmhgfs-only] Error 2
    vmware-iso: make[2]: *** [sub-make] Error 2
    vmware-iso: make[1]: *** [all] Error 2
    vmware-iso: make[1]: Leaving directory `/usr/src/linux-headers-3.2.0-4-amd64'
    vmware-iso: make: *** [vmhgfs.ko] Error 2
    vmware-iso: make: Leaving directory `/tmp/modconfig-VEFxrq/vmhgfs-only'
```

--- a/script/vmtool.sh
+++ b/script/vmtool.sh
@@ -7,12 +7,14 @@ if [[ $PACKER_BUILDER_TYPE =~ vmware ]]; then
     cd /tmp
     mkdir -p /mnt/cdrom
     mount -o loop /home/vagrant/linux.iso /mnt/cdrom
-    tar zxf /mnt/cdrom/VMwareTools-*.tar.gz -C /tmp/
-    /tmp/vmware-tools-distrib/vmware-install.pl -d
-    rm /home/vagrant/linux.iso
+    #tar zxf /mnt/cdrom/VMwareTools-*.tar.gz -C /tmp/
+    git clone https://github.com/rasa/vmware-tools-patches.git
+    vmware-tools-patches/untar-and-patch.sh /mnt/cdrom/VMwareTools-*.tar.gz
+    vmware-tools-distrib/vmware-install.pl -d
     umount /mnt/cdrom
     rmdir /mnt/cdrom
-    rm -rf /tmp/VMwareTools-*
+    rm /home/vagrant/linux.iso
+    rm -rf /tmp/VMwareTools-* /tmp/vmware-tools-patches /tmp/vmware-tools-distrib
 fi
 
 if [[ $PACKER_BUILDER_TYPE =~ virtualbox ]]; then

--- a/script/vmtool.sh
+++ b/script/vmtool.sh
@@ -2,7 +2,7 @@
 
 if [[ $PACKER_BUILDER_TYPE =~ vmware ]]; then
     echo "==> Installing VMware Tools"
-    apt-get install -y linux-headers-$(uname -r) build-essential perl
+    apt-get install -y linux-headers-$(uname -r) build-essential perl git
 
     cd /tmp
     mkdir -p /mnt/cdrom

